src/mol-model-props/computed/representations/interactions-intra-unit-cylinder.ts

```diff
@@ -66,9 +67,11 @@ async function createIntraUnitInteractionsCylinderMesh(ctx: VisualContext, unit:
                 let minDistB = minDistA;
                 Vec3.copy(posA, pA);
                 Vec3.copy(posB, pB);
+                const donorType = t === InteractionType.HydrogenBond ? FeatureType.HydrogenDonor : FeatureType.WeakHydrogenDonor;
+                const isHydrogenDonorA = types[offsets[a[edgeIndex]]] === donorType;
 
-                eachIntraBondedAtom(unit, idxA, (_, idx) => {
-                    if (isHydrogen(structure, unit, elements[idx], 'polar')) {
+                if (isHydrogenDonorA) eachIntraBondedAtom(unit, idxA, (_, idx) => {
```

Comment:
please add curly brances to the if-else statement

Reply:
done